src/control/cmd/dmg/pool.go

```diff
@@ -88,6 +89,36 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return poolDestroy(d.log, d.conns, d.Uuid, d.Force)
 }
 
+// PoolSetPropCmd represents the command to set a property on a pool.
+type PoolSetPropCmd struct {
+	logCmd
+	connectedCmd
+	UUID     string `long:"pool" required:"1" description:"UUID of DAOS pool"`
+	Property string `short:"n" long:"name" required:"1" description:"Name of property to be set"`
+	Value    string `short:"v" long:"value" required:"1" description:"Value of property to be set"`
+}
+
+// Execute is run when PoolSetPropCmd subcommand is activated.
+func (c *PoolSetPropCmd) Execute(_ []string) error {
+	req := client.PoolSetPropReq{
+		UUID:     c.UUID,
+		Property: c.Property,
+	}
+
+	req.SetString(c.Value)
+	if numVal, err := strconv.ParseUint(c.Value, 10, 64); err == nil {
+		req.SetNumber(numVal)
+	}
+
+	if err := c.conns.PoolSetProp(req); err != nil {
+		return errors.Wrap(err, "pool set-prop failed")
+	}
+
+	// TODO: send the set value back in the resp to confirm?
```

Comment:
Does that imply that the handler on the C side should query the property after setting it? I kind went back and forth on this... Seems like we should trust that either it's set properly or we get an error.

Reply:
That's what we do for the ACL modification stuff, we ask for the actual updated value. Even for the overwrite.

Reply:
Ok, I'll follow suit.

Reply:
Hmm. This seems to be turning into A Thing. I need to add a new CaRT RPC to get a pool property?

Reply:
We do need a get-prop at some point, but I think it would be best for someone with more expertise in this area to add it. I'm quickly running out of cycles to finish this task and all of the other stuff on my plate for 0.9.

Reply:
I've already implemented that in #1587. Maybe you could address this piece in a follow-on patch?

Reply:
Oh hey, look at that... I can rebase this PR on that one. Pretty sure that will land before this one.